requests/adapters.py

```diff
@@ -236,6 +238,12 @@ def cert_verify(self, conn, url, verify, cert):
                 conn.key_file = cert[1]
             else:
                 conn.cert_file = cert
+            if conn.cert_file and not os.path.exists(conn.cert_file):
+                raise IOError("Could not find the SSL certificate file, "
+                              "invalid path: {0}".format(conn.cert_file))
+            if conn.key_file and not os.path.exists(conn.key_file):
+                raise IOError("Could not find the SSL key file, "
+                              "invalid path: {0}".format(conn.key_file))
```

Comment:
For new exception strings, can we use the acronym `TLS` instead of the acronym `SSL`? I don't want to change the string in the block above because people may be checking the actual string message, but these ones should use the proper name for the protocol.